tests/js/client/shell/shell-hotbackup-noncluster.js

```diff
@@ -0,0 +1,147 @@
+/*jshint globalstrict:false, strict:false */
```

Comment:
shouldn't this test be in the enterprise repository?

Reply:
Yes, it should, will move, once it works in Jenkins.